cmd/version.go

```diff
@@ -22,8 +22,8 @@ func newVersionCmd(rootCmd *cobra.Command) *cobra.Command {
 	// versionCmd represents the version command
 	var versionCmd = &cobra.Command{
 		Use:   "version",
-		Short: "Show Appsody CLI version",
-		Long:  ``,
+		Short: "Output Appsody CLI version.",
```

Comment:
Displays the version of Appsody CLI?

Reply:
I prefer the word "output" over "display" in this case, but I will change the working from "Output Appsody CLI version" --> "Output the version of the Appsody CLI"

Reply:
I prefer the original "show" (as in "Show the version of the Appsody CLI"), as "output" can be something generated from something else :)

Reply:
@doveye okay, we'll go with that :)